packages/vscode-extension/WHATISNEW.md

```diff
@@ -1,4 +1,29 @@
-## 2.7.0 - Sep 17, 2021
+
+- [2.8.0 - Oct 18 2021](#280---oct-18-2021)
+- [2.7.0 - Sep 17 2021](#270---sep-17-2021)
+
+## 2.8.0 - Oct 18 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Preview Features:
+- Support management of multiple environments.
+- Support project collaborations among multiple developers.
+- Support customize manifest.
+- Improve the experience to customize Azure resource provision using ARM(Azure Resource Manager).
+
+The picture below shows how to enable preview features, more details refer to the [preview guidance](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit)
+
+![Alt Text](https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/img/enable-preview.gif)
+
+Enhancement:
+
+- Improve UI experience and Command Pallette experience. Ensure the same experience between Tree View and Command pallette.
```

Comment:
It is not clear of what the change is? Is the enhancement for "making the command and tree view text be consistent"?